### DISCUSSION TITLE
Update pip3.7 package to OpenSlides 3.4 for Uberspace

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenSlides/OpenSlides#6283] [General]
there have already been considerations to update the pip package to v3.4.? 

@emanuelschuetze @tsiegleauq 

Siehe: https://pypi.org/project/openslides/

**Comment by tsiegleauq** [comment]
I suppose this cannot be done, since docker and the external autoupdate server (written in go) is a hard requirement now

**Reply by linux-lukas** [reply]
Okay. :/ 

Alternatively, how could it be made possible to get OpenSlides running on Uberspaces?

**Reply by tsiegleauq** [reply]
> Okay. :/
> 
> Alternatively, how could it be made possible to get OpenSlides running on Uberspaces?

This? https://uberspace.de/
If they offer you an ubuntu server or debian server, simply install docker and install OpenSlides 3.4. I cannot really tell what they offer, seems they don't give explicit information about their systems. Ask them I suppose.

**Reply by linux-lukas** [reply]
Yes, but docker will not run on an Uberspace. 

Gradually migrating from CentOS 7 to Debian 11, as Uberspace mentioned in a Mastodon post: https://uberspace.social/@ops/107088236207665649

**Reply by tsiegleauq** [reply]
well you **can** run the OS3.4 setup without docker. You would need to setup the OpenSlides (backend) Server, nginx for the client, the os3-autoupdate, the os3-mediafile server and connect them all together via (i.e) caddy (or any other proxy). I would not advice to do that manually (thats why we use docker), but yes, it can work just as well, but I suppose nobody ever dared to try that out :D
